src/System.Management.Automation/engine/ArgumentTypeConverterAttribute.cs

```diff
@@ -152,11 +152,6 @@ internal object Transform(EngineIntrinsics engineIntrinsics, object inputData, b
                         if (_convertTypes[i] == typeof(ActionPreference))
```

Comment:
@KirkMunro, your last commit had 120 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Error occurred in test script 'D:\a\1\s\test\powershell\engine\Api\TypeInference.Tests.ps1'
```powershell
Index was outside the bounds of the array.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\engine\Api\TypeInference.Tests.ps1'
```powershell
Index was outside the bounds of the array.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1'
```powershell
Index was outside the bounds of the array.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 147
```
Error occurred in test script 'D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1'
```powershell
Index was outside the bounds of the array.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 147
```
Error occurred in test script 'D:\a\1\s\test\powershell\Host\Logging.Tests.ps1'
```powershell
Index was outside the bounds of the array.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\Logging.Tests.ps1: line 9
```